### ISSUE TITLE
Highlight Active Line Only Works After Deleting A Line

### ISSUE THREAD

**Issue by d4rkb1ue** [securingsincity/react-ace#190]
`highlightActiveLine` is opened as default. But it doesn't work at beginning. The highlighter only shows after some editing, for example, delete a whole line. 
This problem can be seen from the `react-ace/example`:

1. At beginning, the first line doesn't highlight.
2. Press `Enter` and then press `Delete`
3. The first line become highlighted.

**Comment by brookshi** [comment]
caused by `this.handleMarkers`,
```js
this.handleMarkers(markers || []);
```
may use this code below is a better way
```js
if (markers && markers.length > 0) {
   this.handleMarkers(markers || []);
}
```

**Comment by securingsincity** [comment]
@d4rkb1ue this is fixed in `4.2.2` and should work in the example thanks to @brookshi

**Comment by zheeeng** [comment]
@securingsincity I met this problem again, if I set markers on AceEditor.
